stoiip_prob/main.py

```diff
@@ -24,64 +24,111 @@
 
 print(np.around(stoiip(area, h, poro, swc, boi), 2))
 
-# %% Create random values for each distribution
+# %% Defining random values for porosity from norm_values variable
 
-norm_values = norm.rvs(loc=0.5, scale=0.3, size=1000)
-expon_values = expon.rvs(loc=0, scale=0.3, size=1000)
-lognorm_values = lognorm.rvs(s=0.95, size=1000)
-triang_values = triang.rvs(c=0.3, loc=0, scale=1, size=1000)
+# Normal distribution
+norm_values = norm.rvs(loc=0.2, scale=0.3, size=1000)
```

Comment:
Inconsistent naming compared to the last implementations where you used the same variable name since the beginning i.e. "norm_values" vs "porosity_norm".  See saturation_norm for example, where all the assignations have the same variable which makes sense. Here you named them different

Reply:
done